subxt/examples/frontier_eth.rs

```diff
@@ -0,0 +1,59 @@
+#![allow(missing_docs)]
+
+use subxt::OnlineClient;
+use subxt_signer::eth::{dev, AccountId20, Signature};
+
+#[subxt::subxt(runtime_metadata_insecure_url = "ws://127.0.0.1:9944")]
+mod eth_runtime {}
+
+enum EthRuntimeConfig {}
+
+impl subxt::Config for EthRuntimeConfig {
+    type Hash = subxt::utils::H256;
+    type AccountId = AccountId20;
+    type Address = AccountId20;
+    type Signature = Signature;
+    type Hasher = subxt::config::substrate::BlakeTwo256;
+    type Header =
+        subxt::config::substrate::SubstrateHeader<u32, subxt::config::substrate::BlakeTwo256>;
+    type ExtrinsicParams = subxt::config::SubstrateExtrinsicParams<Self>;
+    type AssetId = u32;
+}
+
+// This helper makes it easy to use our `eth_signer::AccountId20`'s with generated
+// code that expects a generated `eth_runtime::runtime_types::foo::AccountId20` type.
+// an alternative is to do some type substitution in the generated code itself, but
+// mostly I'd avoid doing that unless absolutely necessary.
+fn to_runtime_account_id(a: AccountId20) -> eth_runtime::runtime_types::fp_account::AccountId20 {
+    eth_runtime::runtime_types::fp_account::AccountId20(a.0)
+}
```

Comment:
Could also `impl From`, but for the sake of the example this is easy to follow :)